assignment2.txt

```diff
@@ -0,0 +1,29 @@
+Emma Ladouceur homework2 questions
+
+Question 1
+	Absolutely! Its quite common to mix two operators in equations. For example for the quadratic equation divition multiplication and square roots are all used. The important thing is remebering order of operations and parenthesis
+
+
+Question 2
+	Systax errors involve forgetting a ; or making a mistake in the coding of a program whereas logical errors are issues with the program and how it goes about completing its task.
```

Comment:
Correct - it's also important to note that syntax errors occur at compilation, whereas logical errors occur at runtime.